### DISCUSSION TITLE
changing <Menu> background color in dark mode

### DISCUSSION THREAD

**Discussion by Amir-Zouerami** [chakra-ui/chakra-ui#5326] [Q&A]
I have searched everywhere but i cannot find anything about this. How can I change the background color of the [menus](https://chakra-ui.com/docs/overlay/menu)? I'm using dark mode and I have changed the background color of the <body> with extendTheme():

```tsx
const theme = extendTheme(
  {
    styles: {
      global: (props: StyleFunctionProps) => ({
        body: {
          bg: mode("white", "#202124")(props),
        },
      }),
    },
    config: {
      initialColorMode: "dark",
      useSystemColorMode: false,
    },
    fonts: {
      heading: `IRYEKAN BOLD, ${baseTheme.fonts.heading}`,
      body: `IRYEKAN REG, ${baseTheme.fonts.body}`,
    },
    direction,
  },
  withDefaultVariant({
    variant: "filled",
    components: ["Input", "Select"],
  }),
  withDefaultColorScheme({
    colorScheme: "gray",
  })
);
```
but I cannot find the menu background property.

**Is there a list somewhere? because the types are not all there yet. I'd like to do the same thing for select inputs, popovers and other elements as well.**

**Comment by primos63** [comment]
> I have searched everywhere but i cannot find anything about this. How can I change the background color of the menus?

Did you look at the [theme source](https://github.com/chakra-ui/chakra-ui/blob/main/packages/menu/src/menu.tsx) for `Menu`?

The `Menu` component itself doesn't have a background or styles. The styles the component reads from the theme are placed in a context that can be accessed by its children. I think what you may be looking for is `MenuList`

**Comment by kisbalazspatrik** [comment]
Okay, I had to spend 2 hours trying to fix this but this helped me: https://chakra-ui.com/docs/components/menu/theming
So, right now for me it looks like:

```
const { definePartsStyle, defineMultiStyleConfig } =
    createMultiStyleConfigHelpers(menuAnatomy.keys);

  const baseStyle = definePartsStyle({
    list: {
      bg: "teal.500",
    },
    item: {
      color: "gray.200",
      bg: "teal.500",
      _hover: {
        bg: "teal.600",
      },
      _focus: {
        bg: "teal.600",
      },
    },
  });
  const menuTheme = defineMultiStyleConfig({ baseStyle });

  const components = {
    Menu: menuTheme,
  };

  const theme = extendTheme({ components });
```
Hint: `useColorModeValue` does not work here.

**Comment by JunlinPan-Crypto** [comment]
Use `_dark` and `_light` [pseudo](https://chakra-ui.com/docs/styled-system/style-props#pseudo) and `--menu-bg` to specify menu background:

**theme/menu.ts**
```ts
import { menuAnatomy } from '@chakra-ui/anatomy'
import { createMultiStyleConfigHelpers } from '@chakra-ui/react'

const { definePartsStyle, defineMultiStyleConfig } =
  createMultiStyleConfigHelpers(menuAnatomy.keys)

const baseStyle = definePartsStyle({
  list: {
   // _dark or _light
    _dark: {
      '--menu-bg': 'colors.gray.900',
    },
  },
})

export default defineMultiStyleConfig({ baseStyle })

```
